static/src/javascripts/projects/commercial/modules/dfp/prepare-googletag.js

```diff
@@ -134,14 +150,18 @@ export const init = (): Promise<void> => {
             }
         });
 
+        // Prebid will already be loaded, and window.googletag is stubbed in `commercial.js`.
         // Just load googletag. Prebid will already be loaded, and googletag is already added to the window by Prebid.
-        return loadScript(
-            config.get(
-                'libs.googletag',
-                '//www.googletagservices.com/tag/js/gpt.js'
-            ),
-            { async: false }
-        );
+        if (canRun && !isInTcfv2EnforcementVariant) {
```

Comment:
I think it's cleaner to just set `canRun` above and only have `loadScript` in one place.

Reply:
I understand this is necessary because `onConsentChange` is async.

Reply:
Yap, due to the asynchronous nature of the consent callback we need to have the loading of the script in two different places for tcfv1 and tcfv2